rest/route/service.go

```diff
@@ -104,6 +104,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/task/{task_id}/start").Version(2).Post().Wrap(requireTask, requirePodOrHost).RouteHandler(makeStartTask(env))
 	app.AddRoute("/task/{task_id}/test_logs").Version(2).Post().Wrap(requireTask, requirePodOrHost).RouteHandler(makeAttachTestLog(settings))
 	app.AddRoute("/task/{task_id}/git/patch").Version(2).Get().Wrap(requireTask).RouteHandler(makeGitServePatch())
+	app.AddRoute("/task/{task_id}/git/version").Version(2).Get().Wrap(requireTask).RouteHandler(makeGitServeVersion())
```

Comment:
This should probably be `/task/{task_id}/version` since the route doesn't actually have anything to do with git. (I don't know why the patch route uses `/git/patch` but I assume it's for legacy reasoning that no longer applies.)

Reply:
I removed it for both just to keep consistency/no one adds it by mistake to a new related route